### DISCUSSION TITLE
Changed code signature in version 1.0.1?

### DISCUSSION THREAD

**Discussion by suschizu** [mucommander/mucommander#682] [Q&A]
hey @ahadas - can you confirm that the code signature has changed in macOS version 1.0.1
I've receive the warning:
`EXPECTED Eclipse Foundation, Inc. (JCDTMS22B4) but FOUND Arik Hadas (CZHQQ65L42)`
thanks!

**Comment by ahadas** [comment]
Yes, jpackage signed all the resources within the dmg with `Arik Hadas (CZHQQ65L42)` in order to be able to notarize the dmg

**Reply by suschizu** [reply]
great - thanks for the confirmation :)
